src/framework/analyses.ml

```diff
@@ -360,7 +360,6 @@ sig
   val branch: (D.t, G.t, C.t, V.t) ctx -> exp -> bool -> D.t
   val body  : (D.t, G.t, C.t, V.t) ctx -> fundec -> D.t
   val return: (D.t, G.t, C.t, V.t) ctx -> exp option  -> fundec -> D.t
-  val intrpt: (D.t, G.t, C.t, V.t) ctx -> D.t
```

Comment:
I started thinking now, maybe interrupts (with confusingly named `SelfLoop` edges) is something we should support for standard C and Linux kernel. In standard C, signal handlers are also a form of interrupts AFAIK, because they may arbitrarily get executed at any time and in idiomatic usage modify globals. In Linux kernel, there's an explicit notion of interrupts that our kernel analysis also doesn't handle.

Although adding this transfer function back, if even needed, should be sufficiently easy, so we don't have to keep it around.

Reply:
Good point, I'd however still say that we only add it back when we need it. I'm also not sure if these self loops are the best way to handle interrupts, it would seem that it introduces widening points almost anywhere?

Reply:
I thought the same thing, so `sync` would be a better way to do it.